up-l1/zenoh.adoc

```diff
@@ -93,22 +93,21 @@ Different message types **MUST** use different Zenoh API.
 
 === URI Mapping
 
-UUri **MUST** be able to be serialized into link:../basics/uri.adoc#42-micro-uris[micro form].
-Then we can transform the micro-form UUri, which is `u8` array, into string in lowercase hexadecimal representation.
+UUri **MUST** be able to be serialized into link:../basics/uri.adoc#short-uris[short form].
+Then we can transform the short-form UUri into string for Zenoh key.
 
 Zenoh key for uProtocol consists of three parts and they are separated by `/`.
-Zenoh key **MUST** start with `upl/` or `upr/`.
-While the local UUri maps to `upl/`, the remote UUri maps to `upr/`.
-If UAuthority is defined in UUri, it **MUST** be put in the second part.
-Then the rest of the UUri **MUST** be put in the last part.
 
-To be more specific:
+* `up/`: Zenoh key **MUST** always start with `up/` and the scheme (`up://`) in the short UUri is always removed.
+* UAuthority: UAuthority is not mandatory, and it can be ignored in local UUri. However, it **MUST** always exist in Zenoh key.
```

Comment:
Could you expand on this a bit? I'm having a hard time parsing it out.

Where is UAuthority not mandatory? Within the UURIs in a UMessage when calling `UTransport::send()`? When calling `UTransport::register_listener()`?

And if it's not mandatory, how do we ensure it always exists in the Zenoh key? The constructor which takes a UAuthority?

Reply:
I think I didn't describe it well. How about now?